app/assets/stylesheets/components/_email-signup-section.scss

```diff
@@ -0,0 +1,26 @@
+.app-c-email-signup--with-background-and-border {
```

Comment:
Slight inconsistency here - the component and files are named `Email signup section` but the classes use `email-signup`. Can I suggest `Signup link` instead?